buildSrc/src/main/kotlin/Crates.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+object Crates {
+    val AWS_RUNTIME = listOf(
```

Comment:
I'm not sure about pushing this information into buildSrc, this feels like maybe it belongs in the individual builds? Could be convinced otherwise though

Reply:
I moved it into buildSrc since I needed it from both `aws/sdk` and `rust-runtime`. I prefer to have only one place to update the list.